example-deployment/docker/dashboard.dockerfile

```diff
@@ -19,9 +19,17 @@ FROM nginx:stable
 COPY --from=0 /root/rmf-web/packages/dashboard/build /usr/share/nginx/html/dashboard
 SHELL ["bash", "-c"]
 RUN echo -e 'server {\n\
+  listen 80;\n\
+  server_name localhost;\n\
```

Comment:
( i accidentally deleted the previous comment sorry ) should this be example.com, in the example-deployment context?

Reply:
cancel the above, i think its incorrect. much to learn!